src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
+      rel="stylesheet" 
+      type="text/css">
```

Comment:
```suggestion
    <link 
      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" 
      rel="stylesheet" 
      type="text/css"
    >
```
Closing tag should be just under the opening one

Reply:
I got it